vscode-antimony/src/server/stibium/stibium/tree_builder.py

```diff
@@ -42,6 +49,15 @@
     'namemaybein': NameMaybeIn,
     'reaction_name': ReactionName,
     'reaction': Reaction,
+    'event': Event,
+    # 'event_name': EventName,

```

Comment:
If you're not using EventName, just delete this line instead of commenting it out.